### ISSUE TITLE
No sandy beaches with ATG

### ISSUE THREAD

**Issue by cyres** [Haggle1996/RevolutionPack#147]
I tried very different configurations in the atg-configs but there will be no `Minecraft-Beach`. I also tried configs from other modpacks and removed BiomesOP for testing purpose, `Minecraft-Beach` does not work with this pack. 
But i found a fix in the `generator.gfg` of atg:
```yml
Coast-Beach {
    D:Minecraft-Beach=1.0
    D:"<DragonAPI ASM>-Beach"=1.0
}
```
Will do the thing.

**Comment by NachtRaben** [comment]
As far as I'm aware ATG won't be included in 2.0 of the pack due to the mod author not updating it frequently enough for bugfixes. Haggle might push an update with this for the 1.3 branch though.

**Comment by Haggle1996** [comment]
I had actually disabled sandy beaches a looooooong time ago (like around version 0.3) because ATG was generating huge swaths of them instead of proper deserts.
